target-platform/config/kura.target-platform.build.properties

```diff
@@ -24,7 +24,7 @@ com.google.protobuf.version=2.6.0
 org.tigris.mtoolkit.iagent.rpc.version=3.0.0.20110411-0918
 com.gwt.user.version=1.0.0
 org.hamcrest.core.version=1.1.0.v20090501071000
-jdk.dio.version=1.0.5
+jdk.dio.version=1.0.6-SNAPSHOT
```

Comment:
So we only track the version of the parent module jdk.dio in kura.target-platform.build.properties and we don't individually version the fragments, correct?